x-pack/plugins/monitoring/public/views/apm/overview/index.js

```diff
@@ -23,15 +23,17 @@ uiRoutes.when('/apm', {
     },
   },
   controller: class extends MonitoringViewBaseController {
-    constructor($injector, $scope) {
+    constructor($injector, $scope, i18n) {
       const $route = $injector.get('$route');
       const globalState = $injector.get('globalState');
       $scope.cluster = find($route.current.locals.clusters, {
         cluster_uuid: globalState.cluster_uuid
       });
 
       super({
-        title: 'APM',
+        title: i18n('xpack.monitoring.apm.overviewRouteTitle', {
```

Comment:
According to [Glossary](https://docs.google.com/document/d/1e-lHAPAtxodP8U4gbwA3lw8FyhhW7M303--77nMpe8w/edit) `APM` remains untranslated.